integration/spark/src/main/common/java/io/openlineage/spark/agent/lifecycle/plan/InsertIntoHiveTableVisitor.java

```diff
@@ -42,13 +40,9 @@ public static boolean hasHiveClasses() {
   public List<OpenLineage.Dataset> apply(LogicalPlan x) {
     InsertIntoHiveTable cmd = (InsertIntoHiveTable) x;
     CatalogTable table = cmd.table();
+    String authority = SparkConfUtils.getMetastoreKey(context.getConf()).orElse(null);
```

Comment:
This is usually going to come back as something like `thrift://10.1.0.1:9083` and may even be a comma delimited string. You can't combine that with `hive:` as you do in https://github.com/OpenLineage/OpenLineage/pull/416/files#diff-573381c3efcbea3421e42a897ecdc138b49cbc872c7539c1a5468b1d36a1189dR44 .

Reply:
Fixed this, and added test for those cases: https://github.com/OpenLineage/OpenLineage/pull/417/files#diff-457f6541bb4a2f1a3a5b9877c074dd398c470ba035e3c4d7f3e582db153a32e7R113

As agreed offline, the hive URI is now a primary source for a dataset namespace.